docs/references/javascript/sign-in/authenticate-with.mdx

```diff
@@ -82,14 +103,21 @@ function authenticateWithWeb3(params: AuthenticateWithWeb3Params): Promise<SignI
   - `identifier`
   - `string`
 
-  The user's Web3 ID.
+  The user's public web3 wallet address.
```

Comment:
I see you added "public" here and use it often in other places. Does "public" need to be added to all instances of "web3 wallet address" ?

Reply:
Good question! No i believe we can omit "public" as a web3 wallet address will be always public. I will update the rest of the places as well to be aligned and use a single term